theme/static/js/hs-vue/left-header-app.js

```diff
@@ -0,0 +1,43 @@
+let leftHeaderApp = new Vue({
+    el: '#left-header-table',
+    delimiters: ['${', '}'],
+    data: {
+        owners: USERS_JSON.map(function(user) {
+            user.loading = false;
+            return user;
+        }).filter(function(user){
+            return user.access === 'owner';
+        }),
+        userCardSelected: {
+            user_type: null,
+            access: null,
+            id: null,
+            pictureUrl: null,
+            best_name: null,
+            user_name: null,
+            can_undo: null,
+            email: null,
+            organization: null,
+            title: null,
+            contributions: null,
+            subject_areas: null,
+            identifiers: [],
+            state: null,
+            country: null,
+            joined: null,
+        },
+        lastChanagedBy: LAST_CHANGED_BY,
+        cardPosition: {
+            top: 0,
+            left: 0,
+        }
+    },
+    methods: {
+        onLoadOwnerCard: function(data) {
+            let el = $(data.event.target);
+            this.userCardSelected = data.user;
+            this.cardPosition.left = el.position().left - 175 + el.width() / 2;
+            this.cardPosition.top = el.position().top + 30;
+        }
```

Comment:
I genuinely don't know here so totally just asking -- is this the best way to position this with pixel math?

Reply:
In this case, yes. Because the profile card is absolute-positioned. 

I could have written this a bit clearer, but this is just a simple centering formula. 175px is equivalent to half the card's width (350px / 2) and the 30px are a visual margin. I will clarify this with constants in my next PR as part of this work.

Reply:
Then why are you both computing the 175 as well as hardcoding it here?  `el.width() / 2` and `- 175`?

Reply:
and isn't that effectively adding zero?

Reply:
The `el` variable here represents the link that has been clicked, not the card. So the logic is using its position and the width of the card (175px) to calculate where it needs to be positioned so it's centered.